terraform/gcp/projects/pangeo-hubs.tfvars

```
@@ -2,7 +2,7 @@ prefix                 = "pangeo-hubs"
 project_id             = "pangeo-integration-te-3eea"
 zone                   = "us-central1-b"
 region                 = "us-central1"
-core_node_machine_type = "n1-highmem-4"
+core_node_machine_type = "n1-highmem-8"
```

I suggest sticking with n2 series at all times as they provide a robust 1:8 ratio of CPU:RAM while the n1 series is an exception to the ratio of 1:8, providing only 1:6.5. The cost increase is 10% but the RAM increase is 23%, and CPU performance appears to be ~20-30% faster as well.

```suggestion
core_node_machine_type = "n2-highmem-8"
```

There is a case to be made that if we never need to bump this again that its better to stay with the slightly cheaper n1 machine so you can merge without going n2, but I think we should stick with n2 by default when choosing.